Classes/Plugin/OaiPmh.php

```diff
@@ -1018,18 +1041,21 @@ protected function generateOutputForDocumentList(DocumentList $documentListSet)
      */
     protected function generateResumptionTokenForDocumentListSet(DocumentList $documentListSet)
     {
-        if ($documentListSet->count() != 0) {
-            $token = uniqid();
-            $GLOBALS['TYPO3_DB']->exec_INSERTquery(
-                'tx_dlf_tokens',
-                [
+        if ($documentListSet->count() !== 0) {
+            $token = uniqid('', false);
```

Comment:
Is this just for clarity or why do you explicitly set the default parameters?

Reply:
Thats a suggestion from phpstorm for uniqid().